lib/internal/per_context/primordials.js

```diff
@@ -51,7 +77,13 @@ function copyPropsRenamed(src, dest, prefix) {
     if ('get' in desc) {
       copyAccessor(dest, prefix, newKey, desc);
     } else {
-      ReflectDefineProperty(dest, `${prefix}${newKey}`, desc);
+      const name = `${prefix}${newKey}`;
+      ReflectDefineProperty(dest, name, desc);
+      if (varargsMethods.includes(name)) {
+        ReflectDefineProperty(dest, `${name}Apply`, {
+          value: applyBind(desc.value, src),
```

Comment:
```suggestion
          value: applyBind(desc.value),
```

Reply:
This is done to ensure that the method stays `static`‑like.

Reply:
Gotcha! I think we should test those to ensure the implementation is correct – and stays correct.

Should it be `null` instead of `src` or it doesn't matter?
```suggestion
          value: applyBind(desc.value, null),
```
Alternatively, we could add a comment explaining why we put `src` as second parameter.